flashplugin-x86_64.lkgv

```diff
@@ -0,0 +1,3 @@
+VERSION=23.0.0.185
+URL=https://fpdownload.adobe.com/pub/flashplayer/pdc/23.0.0.185/flash_player_ppapi_linux.x86_64.tar.gz
```

Comment:
Please use the VERSION variable here so we only have to type the new version number in one place